hugo/content/memory/_index.md

```diff
@@ -35,69 +33,74 @@ The first 2 bits in a 32-bit address.
+| `TK1_BLAKE2S`       | 0xff000040     | r/w  | r     | Function pointer to a BLAKE2S function in the firmware.                                                                                                                                                               |
+| `TK1_CDI_FIRST`     | 0xff000080     | r/w  | r     | The computed Compound Device Identifier (CDI). 8 words.                                                                                                                                                               |
+| `TK1_CDI_LAST`      | 0xff00009c     | r/w  | r     | Last word of CDI.                                                                                                                                                                                                     |
+| `TK1_UDI_FIRST`     | 0xff0000c0     | r    | r     | First word of Unique Device ID (UDI). 2 words. Set during provisioning.                                                                                                                                               |
+| `TK1_UDI_LAST`      | 0xff0000c4     | r    | r     | Last word of UDI                                                                                                                                                                                                      |
+| `TK1_RAM_ADDR_RAND` | 0xff000100     | r/w  | r/w   | Seed word for the RAM address randomization.                                                                                                                                                                          |
+| `TK1_RAM_SCRAMBLE`  | 0xff000104     | r/w  | r/w   | Seed word for the RAM data scrambling.                                                                                                                                                                                |
+| `TK1_CPU_MON_CTRL`  | 0xff000180     | w    | w     | Bit 0 enables CPU execution monitor. Can't be unset. Locks addresses set in `TK1_CPU_MON_FIRST` and `TK1_CPU_MON_LAST`.                                                                                               |
+| `TK1_CPU_MON_FIRST` | 0xff000184     | w    | w     | First adress (XXX word?) of the RAM area monitored for execution attempts.                                                                                                                                            |
+| `TK1_CPU_MON_LAST`  | 0xff000188     | w    | w     | Last address (XXX word?) of the area monitored for execution attempts.                                                                                                                                                |
+
```

Comment:
Should we add SPI-things her as optional?

Reply:
The SPI is still an unmerged feature. This is a change to the Dev Handbook to fix bugs in the memory map. 

This will go live on https://dev.tillitis.se/ the moment we merge.

Reply:
Ok. Good point.